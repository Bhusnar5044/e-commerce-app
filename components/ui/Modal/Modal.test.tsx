import { render } from '@test-utils';
import { Modal } from './Modal';

describe('Modal', () => {
    it('should render properly', () => {
        const { container } = render(
            <Modal open={false} onClose={jest.fn}>
                Dummy
            </Modal>,
        );
        expect(container).toMatchSnapshot();
    });
});
