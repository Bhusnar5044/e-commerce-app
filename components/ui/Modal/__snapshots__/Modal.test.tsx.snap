// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal should render properly 1`] = `<div />`;
